src/app/cases/page.tsx

```diff
@@ -48,64 +57,67 @@ export default function Page() {
       <H1>Browse Available Cases</H1>
       <FiltersContainer>
         <FilterDropdown
-          defaultValue="Remote/In Person"
-          options={['Remote/In Person', 'Remote Only', 'In Person Only']}
-          onChange={v =>
-            setCaseFilters({ ...caseFilters, remote: v as string })
-          }
+          defaultValue={defaultFilterValues.remote}
+          multi
+          options={new Set(['Remote', 'In Person'])}
+          value={caseFilters.remote}
+          onChange={v => setCaseFilters({ ...caseFilters, remote: v })}
         />
         <FilterDropdown
-          defaultValue="Attorney/Interpreter"
-          options={['Attorney/Interpreter', 'Interpreter Only']}
-          onChange={v => setCaseFilters({ ...caseFilters, role: v as string })}
+          defaultValue={defaultFilterValues.role}
+          multi
+          options={new Set(['Interpreter', 'Attorney'])}
+          value={caseFilters.role}
+          onChange={v => setCaseFilters({ ...caseFilters, role: v })}
         />
+        {/* <FilterDropdown
+          defaultValue={defaultFilterValues.languages}
+          multi
+          options={new Set(caseData.flatMap(c => c.languages))}
+          value={caseFilters.languages}
+          onChange={v => setCaseFilters({ ...caseFilters, languages: v })}
+        /> */}
         <FilterDropdown
-          defaultValue="Anywhere"
-          // requires knowing user location
-          options={[
-            'Within 5 miles',
-            'Within 10 miles',
-            'Within 20 miles',
-            'Within 100 miles',
-            'Anywhere',
-          ]}
-          onChange={v =>
-            setCaseFilters({ ...caseFilters, distance: v as string })
-          }
+          defaultValue={defaultFilterValues.agency}
+          multi
+          options={new Set(['Court', 'USCIS'])}
+          value={caseFilters.agency}
+          onChange={v => setCaseFilters({ ...caseFilters, agency: v })}
         />
         <FilterDropdown
-          defaultValue="All countries"
+          defaultValue={defaultFilterValues.countries}
           multi
-          options={[
-            'All countries',
-            ...Array.from(
-              new Set(
-                caseData
-                  .filter(c => c.country)
-                  .map(c => (c.country ? c.country : '')),
-              ),
-            ),
-          ]}
-          onChange={v =>
-            setCaseFilters({ ...caseFilters, countries: v as string[] })
+          options={
+            new Set(
+              caseData.filter(c => c.country).map(c => c.country),
```

Comment:
Can we only pull cases that need either an attorney or an interpreter? I think some of the cases that are initially displayed are not seeking either role, which we don't want to display at all.

Reply:
Do we want to update the query for that or get rid of them entirely from the database?

Reply:
let's update the query!